andres-uzeda/homework7/mvc_task/src/ArrayListModelData.java

```diff
@@ -0,0 +1,45 @@
+import java.util.ArrayList;
+
+public class ArrayListModelData {
```

Comment:
It would be a better idea to inherit from ArrayList  than have an ArrayList Property